website/content/docs/commands/status.mdx

```diff
@@ -49,3 +49,21 @@ flags](/vault/docs/commands) included on all commands.
 - `-format` `(string: "table")` - Print the output in the given format. Valid
   formats are "table", "json", or "yaml". This can also be specified via the
   `VAULT_FORMAT` environment variable.
+
+By default, the output is displayed in "table" format.
+
+#### Output Fields
+
+1. The field for unseal nonce is displayed as `"n"` instead of `n` in yaml outputs. 
+2. The following fields in "table" format are displayed only when relevant:
+- "Unseal Progress" and "Unseal Nonce" are displayed when vault is sealed.
+- "HCP Link Status" and "HCP Link Resource ID" are displayed when HCP link is configured.
+- "Seal Migration in Progress" is displayed when it is in progress.
+- "Cluster Name" and "Cluster ID" are displayed if they have a value 
```

Comment:
There's some inconsistency with punctuation here - the above line ends with a full stop, and this line doesn't - I think we should align on having a full stop on each line.

Reply:
I agree!! I will fix this